### ISSUE TITLE
Console Errors while running unit tests

### ISSUE THREAD

**Issue by markusguenther** [neos/neos-ui#3288]
### Description

While running the unit tests some console errors occur. That did not happen before the esbuild change.
We should investigate this and as us why the tests not failing when there are errors in the execution.

### Steps to Reproduce

1. choose packages/neos-ui-sagas/ for instance
2. yarn test

#### Expected behavior

Test pass without console error.

#### Actual behavior

<img width="1357" alt="Screenshot 2022-12-02 at 22 30 27" src="https://user-images.githubusercontent.com/1014126/205395336-3399c2da-643a-4b2c-8c4b-c3af79d77a92.png">

### Affected Versions

Neos: 8.2.0

UI: 8.2.0

**Comment by markusguenther** [comment]
Duplicate of https://github.com/neos/neos-ui/issues/3258 but with more details ;)

**Comment by mhsdesign** [comment]
yes indeed a dub of https://github.com/neos/neos-ui/issues/3258 :D - i discovered, it has not much to do with the esbuild change but rather since we updated to yarn3
